contents/handbook/growth/marketing/content.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Content
+sidebar: Handbook
+showTitle: true
+---
+
+Regularly publishing useful content helps us:
+
+* Build awareness of PostHog in the developer community
+* Convert people who are aware of us into users
+* Convert users into power users
+
+## Content Personas
+
+We tailor our content to specific audiences.
+
+### Raising awareness
+
+Many developers have no idea who we are, and do not have a current need for PostHog at this time. However, they may develop a need in future. These people are interested in broader content:
+
+* Syndicating content about how to make your product successful - being a pundit.
+  * New and useful open source libraries to build / understand user behavior
+  * Syndicating content about talking to users and how to help engineers become better at product-thinking
+  * Running an engineering team that ships
+  * Big new releases in popular libraries that people use to ship with
+  * Useful new products that are released
+* 1st hand content about building an open source company
+  * Building in the open
```

Comment:
* Stories from other companies

Reply:
I'll make the additon 👍  see comment above.